samples/powerplay/src/main/cpp/PowerPlayMultiPlayer.cpp

```diff
@@ -109,8 +109,17 @@ void PowerPlayMultiPlayer::triggerDown(int32_t index, oboe::PerformanceMode perf
             return;
         }
     }
+
+    mSampleSources[index]->setPlayMode(performanceModeChanged);
```

Comment:
Shouldn't this be reversed? We want to start from the beginning of the stream if the performance mode has changed and continue from where we originally were if we just press the stop button and the start button?

Reply:
Hmmmm I guess this is more of a preference thing for this one. I like the fact the when the performance mode changes, we start playback from the beginning. But that doesn't need to be the case if we don't want it too.

I doing a more thorough code rewrite this this method.